lib/gear_application/config_getter.ex

```diff
@@ -27,6 +27,32 @@ defmodule Antikythera.GearApplication.ConfigGetter do
 
   @doc false
   def get_all_env(gear_name) do
+    case load_config_from_process_dictionary(gear_name) do
+      {:ok   , config              } -> config
+      {:error, current_gear_configs} ->
+        config = load_config_from_ets(gear_name)
+        store_config_to_process_dictionary(gear_name, current_gear_configs, config)
+        config
+    end
+  end
+
+  defp load_config_from_process_dictionary(gear_name) do
+    case Process.get(:gear_configs) do
```

Comment:
Let's make this atom a module attribute.
```ex
@key :gear_configs
```

Reply:
fixed in 40400f1f530816cdb00b656fc408cec823d826e1